CWP/papers/HSF-CWP-2017-04_doma/latex/hsf-cwp-dm.tex

```diff
@@ -306,15 +306,15 @@ \subsubsection{Object and Cloud Storage
 
 Decisions on granularity and structure have the potential of broadening
 the scope of technologies that can be used to store the data; for
-example, exploiting object stores (such as AWS S3) as event stores. In
-addition, this would support parallel reading and writing of events as
+example, exploiting object stores (such as AWS S3) as event stores. Such an approach
+would support parallel reading and writing of events as
 event objects from applications. This could allow applications to better
-scale to large numbers of CPU core where now these applications
+scale to large numbers of CPU cores whereas in current models application
 concurrency is limited by the requirement to sequentially write events
 to files. Currently, the efficiency of of reading and writing events
-from object stores is sensitive to the size of the event. The smaller
-the event the worse is the efficiency.. Some of this may be mitigated by
-creating event bundles (i.e. several events in one object). The
+from object stores is sensitive to the size of the event---
+the smaller the event, the worse the efficiency is. Some of these potential inefficiencies
+may be mitigated by creating event bundles (\emph{i.e.} several events in one object). The
```

Comment:
Generally I would suggest that you do not italicise Latinate abbreviations (most modern English style guides suggest this). If you do want to adopt this style, however, you'd better do it consistently. There definitely are some "e.g."s you'd need to fix.